gtsfm/frontend/detector_descriptor/superpoint.py

```diff
@@ -49,27 +51,23 @@ def detect_and_describe(self, image: Image) -> Tuple[Keypoints, np.ndarray]:
         model.eval()
         # TODO: fix inference issue #110
 
+        # Compute features.
```

Comment:
I think the comments you added here are too verbose.

Reply:
I think it's useful for users that are unfamiliar with SuperPoint. It's not obvious what's being done in that code block to the untrained eye.